### ISSUE TITLE
Win11: NVDA is not announcing anything after adding a New Desktop in the Task View  

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#12759]
### Steps to reproduce:
1. Press Windows+Tab to open Task View experience
2. Tab to New Desktop button
3. Press Enter

### Actual behavior:
The new desktop is created but NVDA does not read a confirmation of action.

### Expected behavior:
NVDA confirms that  new desktop is created and reads the details of it. This works as expected with Narrator and JAWS.

#### NVDA installed/portable/running from source:
Installed/portable

#### NVDA version:
2021.1

#### Windows version:
Windows 11 Insider builds

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No. Narrator and JAWS work as expected.

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, try all this after installing Windows app Essentials add-on. @JCSTeh, I’m seriously considering picking up the virtual desktop announcement issue. Thanks.

**Comment by seanbudd** [comment]
@josephsl - is this something you have already fixed in "Windows app Essentials" that we can use?

**Comment by josephsl** [comment]
Hi, part of that is virtual desktops announcement (#5641), stalled due to a possibility where users may create desktops with identical names next to each other. A simpler implementation is present in Windows App Essentials where only name change event on desktop window is handled, whereas Jamie’s solution is expanding event handler o check for possible desktop name changes whenever you move focus. Thanks.

**Comment by seanbudd** [comment]
I'm removing this from our 2021.3 milestone as this is an old, Win 10, and relatively minor issue.

We are also closing this as a duplicate of #5641.
